src/extension/Pty.ts

```diff
@@ -68,7 +78,7 @@ export default class Pty {
       let waitingForNl = command.split("\n").length;
       let firstData = true;
       let ps1State = 0;
-      let ps1NextCallback = () => {};
+      let ps1NextCallback = () => { };
```

Comment:
Why do you want to add a whitespace here?

Reply:
I think my formater did this for me